components/automate-ui/src/app/components/projects-dropdown/projects-dropdown.component.html

```diff
@@ -1,51 +1,9 @@
-<label>
```

Comment:
All of the deleted text is now the resource-dropdown component; this component subclasses that.